cloud/aws/variables.tf

```diff
@@ -123,7 +123,6 @@ variable "included_services" {
     "AWS/EKS",
     "AWS/ElastiCache",
     "AWS/ElasticBeanstalk",
-    "AWS/ElasticInterface",
```

Comment:
It was apparently a typo for `ElasticInference`, cf. https://github.com/splunk/public-o11y-docs/commit/12407dfccd43e523816e9bba4391b34d254b749f

Reply:
It was a typo in Splunk's doc.

Reply:
Got it, see 6b24b62